<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-11-11T11:31:25Z" name="Default suite" finished-at="2020-11-11T11:31:57Z" duration-ms="31940">
    <groups>
    </groups>
    <test started-at="2020-11-11T11:31:25Z" name="Default test" finished-at="2020-11-11T11:31:57Z" duration-ms="31940">
      <class name="spicetag.testcases.CreateNewEmpTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:spicetag.testcases.CreateNewEmpTest@1e1a0406]" started-at="2020-11-11T17:01:25Z" name="setup" finished-at="2020-11-11T17:01:47Z" duration-ms="21791" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="ValidateEmpType()[pri:1, instance:spicetag.testcases.CreateNewEmpTest@1e1a0406]" started-at="2020-11-11T17:01:47Z" name="ValidateEmpType" finished-at="2020-11-11T17:01:56Z" duration-ms="9196" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at spicetag.testcases.CreateNewEmpTest.ValidateEmpType(CreateNewEmpTest.java:39)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateEmpType -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:spicetag.testcases.CreateNewEmpTest@1e1a0406]" started-at="2020-11-11T17:01:56Z" name="tearDown" finished-at="2020-11-11T17:01:57Z" duration-ms="882" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- spicetag.testcases.CreateNewEmpTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
